.gitignore

```diff
@@ -1,3 +1,6 @@
+# config file
+*.env
```

Comment:
I'd change this to:
`.env*`
`!.env.sample`

This will ignore all .env files but allow a .env.sample file which we should store all the variable names in. 

There should be a  separate sections marked under the .env.sample file with comments. Or something that captures the spirit in the example below.

```
# dev environment vars

# staging environment vars

# local environment vars
```

Reply:
Good idea! Thanks, Gary!

Reply:
I do remember this being something we brought up previously, thanks for the reminder!